ros_buildfarm/sourcerpm_job.py

```diff
@@ -55,6 +55,22 @@ def build_sourcerpm(
         '--depth', '1', '--no-single-branch',
         release_repository_url, os_pkg_name]
 
+    # HACK FOR RHBZ#1960791
+    # RHEL 8.4 shipped a cmake package which omitted the 'cmake3' virtual
+    # package. This hack replaces dependencies on 'cmake3' with 'cmake>3',
```

Comment:
Does `>3` work rather than `>=3` because the minimum package version is strictly greater than 3.0.0 or does `>3` in spec format imply greater than or equal to?

Reply:
68e585c